lib/Dependency.js

```diff
@@ -2,43 +2,45 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-function Dependency() {
-	this.module = null;
-}
-module.exports = Dependency;
+"use strict";
+const compareLocations = require("./compareLocations");
 
-Dependency.prototype.isEqualResource = function(other) {
-	return false;
-};
+module.exports = class Dependency {
+	constructor() {
+		this.module = null;
+	}
 
-// Returns the referenced module and export
-Dependency.prototype.getReference = function() {
-	if(!this.module) return null;
-	return {
-		module: this.module,
-		importedNames: true, // true: full object, false: only sideeffects/no export, array of strings: the exports with this names
+	isEqualResource() {
+		return false;
 	}
-};
 
-// Returns the exported names
-Dependency.prototype.getExports = function() {
-	return null;
-};
+	// Returns the referenced module and export
+	getReference() {
+		if(!this.module) return null;
+		return {
+			module: this.module,
+			importedNames: true, // true: full object, false: only sideeffects/no export, array of strings: the exports with this names
+		};
+	}
 
-Dependency.prototype.getWarnings = function() {
-	return null;
-};
+	// Returns the exported names
+	getExports() {
+		return null;
+	}
 
-Dependency.prototype.updateHash = function(hash) {
-	hash.update((this.module && this.module.id) + "");
-};
+	getWarnings() {
+		return null;
+	}
 
-Dependency.prototype.disconnect = function() {
-	this.module = null;
-};
+	updateHash(hash) {
+		hash.update((this.module && this.module.id) + "");
+	}
 
-Dependency.compare = function(a, b) {
-	return Dependency.compareLocations(a.loc, b.loc);
-};
+	disconnect() {
+		this.module = null;
+	}
 
-Dependency.compareLocations = require("./compareLocations");
```

Comment:
This is technically an API change; nothing left around that assumes it can find `compareLocations` in the constructor?